packages/now-cli/src/util/get-team-by-id.ts

```diff
@@ -1,7 +1,18 @@
 import Client from './client';
-import getTeams from './get-teams';
+import { Team } from '../types';
 
-export default async function getTeamById(client: Client, teamId: string) {
-  const teams = await getTeams(client);
-  return teams.find(team => team.id === teamId) || null;
+const teamCache = new Map<string, Team>();
+
+export default async function getTeamById(
+  client: Client,
+  teamId: string
+): Promise<Team> {
+  if (teamCache.has(teamId)) {
+    return teamCache.get(teamId)!;
+  }
+
+  const team = await client.fetch<Team>(`/teams/${teamId}`);
+  teamCache.set(teamId, team);
+
+  return team;
```

Comment:
I recently changed this in #4071 but it hasn't shipped to stable yet. It also has caching.

I think @lucleray mentioned that client.fetch() was failing on Windows for some reason 🤔

Reply:
@styfle It looks like you only changed `getTeams`, this changes `getTeamById`, which is currently using a list of all teams to get the one, which is way less efficient than this if the cache is empty.

Reply:
Not sure about the `client.fetch` thing on Windows

Reply:
This is the PR that fixed a bug on Windows: https://github.com/zeit/now/pull/3740/files

Reply:
> It looks like you only changed getTeams, this changes getTeamById, which is currently using a list of all teams to get the one

Right, but before you PR `getTeamById()` called `getTeams()` so it would cache all the teams once. I guess it depends on the code-path for a given command if it will use `getTeamById()` and `getTeams()`. I'm find with this solution as long as it works on Windows.

Reply:
Can we make sure this works on windows? 😄